tests/cross-compile/c-cross-compile.slang.expected

```diff
@@ -0,0 +1,110 @@
+result code = 0
```

Comment:
I'm a little unclear on why we have an HLSL output here as the expected output of `c-cross-compile`.